stravalib/model.py

```diff
@@ -926,10 +1241,35 @@ class SubscriptionCallback(
     hub_verify_token: Optional[str] = None
     hub_challenge: Optional[str] = None
 
+    # TODO: error: Required dynamic aliases disallowed  [pydantic-alias]
     class Config:
         alias_generator = lambda field_name: field_name.replace("hub_", "hub.")
 
-    def validate(self, verify_token=Subscription.VERIFY_TOKEN_DEFAULT):
+    # TODO: not sure if this docstring and typing is right- mypy is telling me
+    # there is an inconsistency -  Signature of "validate" incompatible with supertype "BaseModel"  [override]
+    def validate(
```

Comment:
stravalib/model.py:1250: error: Signature of "validate" incompatible with supertype "BaseModel"  [override]
stravalib/model.py:1250: note:      Superclass:
stravalib/model.py:1[25](https://github.com/stravalib/stravalib/actions/runs/5719816139/job/15498500654?pr=387#step:11:26)0: note:          @classmethod
stravalib/model.py:1250: note:          def validate(cls, value: Any) -> SubscriptionCallback
stravalib/model.py:1250: note:      Subclass:
stravalib/model.py:1250: note:          def validate(self, verify_token: str = ...) -> None